.github/workflows/ci-lint-examples.yml

```
@@ -0,0 +1,21 @@
+name: Linting Examples
+
+on: [push]
```

```suggestion
on: [push, pull_request]
```
If the set event is `push` the workflow will be triggered by push events in the same repo. As a result a pull request from a fork will not trigger this workflow.

See https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#about-workflow-events